docs/migration/01_migration_guide_to_v0.5.rst

```diff
@@ -55,23 +55,14 @@ an extension that allows users to access the eigen states.
 
 The following is the corresponding table.
 
-+-------------------------------------+--------------------------------+
-| Former algorithm                    | New algorithm                  |
-+=====================================+================================+
-| ``qis                               | ``qiskit.                      |
```

Comment:
I rewrote the table with cvs-table because the wrong line breaks generates redundant spaces.